### DISCUSSION TITLE
Informed consent with custom button-response options

### DISCUSSION THREAD

**Discussion by joseluisulloafulgeri** [jspsych/jsPsych#2206] [Q&A]
Hi all,

I know we can customize the language for the instructions and the text in the button-response options,
I was wondering if it could be possible to do the same for the informed consent,

Any hint on how do this would be really appreciated,

best,
Jose

**Comment by jodeleeuw** [comment]
Hi Jose,

What informed consent are you referring to? Are you using a particular platform that has informed consent built in?

**Reply by joseluisulloafulgeri** [reply]
Hi Josh,
Sorry for the ambiguity in my question. I'm using Cognition.run and in this platform there is a place to include a informed consent. I realize now this is not directly related to JSPsych. So, is probably the wrong place to ask the question. However, I guess one could create a informed consent using JSPsych to elaborate a informed consent, right?

**Reply by jodeleeuw** [reply]
Yep, absolutely! The `html-button-response` plugin is a good option for this. You can make an HTML-formatted document for your consent agreement, and then have buttons at the bottom for various options. If someone declines, you could end the experiment with `jsPsych.endExperiment()`. Or you could have only an agree button and ask the participant to close the window if they do not want to consent. This is my typical approach, but I know different IRBs have different guidelines about what is allowed.
